docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -73,6 +73,22 @@ const sessionSigs = await litNodeClient.getSessionSigs({
 });
 ```
 
+:::note
+ If running the sdk in a Server enviorment session signatures may not be cached unless you provide an instance of `Storage` to the runtime.
+ [here](https://www.npmjs.com/package/node-localstorage) is an implementation of `LocalStorage` which creates local files to persist storage data.
+ If storage is not available, session keys *MUST* be persisted in an external data store. 
+ ```javascript
+ const LocalStorage = require('node-localstorage').LocalStorage;
+ const litNodeClient = new LitNodeClient({
+  litNetwork: "cayenne",
+  debug: true,
+  storageProvider: {
+    provider: new LocalStorage('./storage.test.db'),
```

Comment:
Does it strictly require a db file. I don't see this in https://www.npmjs.com/package/node-localstorage

Reply:
No i just chose `.db` to highlight it will store what would have been put into the `localstorage` database.
I can rename it if it is confusing

Reply:
I think that'll be better as it's less suggestive